src/cli.rs

```diff
@@ -22,9 +22,9 @@ pub(crate) struct Cli {
     #[clap(long, short, default_value = "10000000")]
     pub(crate) size: u64,
     /// The public key of your Atlas API key.
-    #[clap(long)]
-    pub(crate) public: String,
+    #[clap(long, env)]
+    pub(crate) atlas_public_key: String, // --atlas-public-key or ATLAS_PUBLIC_KEY env var
     /// The private key of your Atlas API key.
-    #[clap(long)]
-    pub(crate) private: String,
+    #[clap(long, env)]
+    pub(crate) atlas_private_key: String, // --atlas-private-key or ATLAS_PRIVATE_KEY env var
```

Comment:
I would not add comments, rather make it more explizit code wise, like this:

```rust
    #[clap(long, env = "ATLAS_PRIVATE_KEY")]
    pub(crate) atlas_private_key: String,
```

Reply:
Done with https://github.com/maoertel/mongodb-ftdc/pull/11/commits/90df26fac894dca995a457a100fee7f69b67a9b9